providers/commons

```diff
@@ -5,7 +5,7 @@
 
 # Azure Storage Account Access Key (so far they have this format)
 # alternatively we can grep for endpoint suffixes like core.windows.net
-[A-Za-z0-9+\/]{86}==
+AccountKey=[A-Za-z0-9+\/=]{88};
```

Comment:
I think this weakens the test a lot. Consider the following environment config:
```sh
COSMOSDB_KEY=TEST3bpK0eC/LMP8FtweEFJysEPJZe4sVJX0zQla3ec/MNUFKklsyW3GrTwoJ3EbXnneSzS3ghwz6czDXYqxeU==
COSMOSDB_NAME=dbname
CUSTOMCONNSTR_COSMOSDB_KEY=AccountEndpoint=$COSMOSDB_URI;AccountKey=$COSMOSDB_KEY;
```
The new regex doesn't catch it although this is valid code (actually, that's my own configuration 😬 ).